src/Foundation/Models/Observable.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Foundation\Models;
+
+use Illuminate\Support\Str;
+use KoalaFacade\DiamondConsole\Enums\Layer;
+use KoalaFacade\DiamondConsole\Models\Model;
+
+/**
+ *@mixin Model
+ */
+trait Observable
+{
+    /**
+     * @var array<int, string>
+     */
+    protected static array $additionalObservers = [];
```

Comment:
i think $observers is enough, and we can use unique array to prevent duplicate observer